mne/io/brainvision/brainvision.py

```diff
@@ -184,16 +194,22 @@ def _read_vmrk_events(fname, response_trig_shift=0):
     events = []
     for info in items:
         mtype, mdesc, onset, duration = info.split(',')[:4]
+        onset = int(onset)
+        duration = int(duration)
         try:
             trigger = int(re.findall('[A-Za-z]*\s*?(\d+)', mdesc)[0])
-            if mdesc[0].lower() == 's' or response_trig_shift is not None:
-                if mdesc[0].lower() == 'r':
-                    trigger += response_trig_shift
-                onset = int(onset)
-                duration = int(duration)
-                events.append((onset, duration, trigger))
         except IndexError:
+            trigger = None
             pass
+        if mtype.lower().startswith('response'):
+            if response_trig_shift is not None:
+                trigger += response_trig_shift
+            else:
+                trigger = None
+        if mdesc in event_id:
```

Comment:
if `event_id` is None here (which I think it can be by docstring description) this will create an error